Source/WebCore/css/process-css-properties.py

```diff
@@ -1125,41 +1130,34 @@ def aliases(self):
         return self.codegen_properties.aliases
 
 
-class Descriptors:
-    def __init__(self, kind, descriptor_method_name_modifier, descriptors):
-        self.kind = kind
-        self.descriptor_method_name_modifierdescriptor_method_name_modifier = descriptor_method_name_modifier
+class DescriptorsSet:
```

Comment:
Is it really a DescriptorsSet and not a DescriptorSet?

Reply:
You are right. `DescriptorSet` would be a better name. Will update.